cli/cmd/inject.go

```diff
@@ -201,6 +212,13 @@ func injectPodTemplateSpec(t *v1.PodTemplateSpec) enhancedPodTemplateSpec {
 		},
 	}
 
+	controlPlaneDNSName := fmt.Sprintf("tcp://proxy-api.%s.svc.cluster.local:%d", controlPlaneNamespace, proxyAPIPort)
```

Comment:
1. This is not a DNS name, it's a URL, so `controlPlaneDNSName` isn't the right name to use as the code is now.

2. We shouldn't duplicate the logic for formatting the "tcp://" prefix and and port number here and below. We should format the port number and the URL in one place.